t/50reverse-proxy-session-resumption.t

```diff
@@ -130,13 +130,16 @@ EOC
 subtest 'reproxy' => sub {
     my $upstream = create_upstream();
     my $upstream_port = $upstream->{tls_port};
-    my $app_port = empty_port();
-    my $app_server = spawn_server(
-        argv => [ "plackup -s Standalone --port $app_port -e \"sub { [200, ['X-Reproxy-URL' => 'https://127.0.0.1:$upstream_port/'], []] }\"" ],
-        is_ready => sub {
-            check_port($app_port);
-        },
-    );
+    my $app_server = spawn_h2o(<< "EOT");
+num-threads: 2
+hosts:
+  default:
+    paths:
+      /:
+        header.add: "X-Reproxy-URL: https://127.0.0.1:$upstream_port"
+        file.dir: @{[ DOC_ROOT ]}
+EOT
+    my $app_port = $app_server->{port};
```

Comment:
Or maybe use `plackup -s Starlet`? My understanding is that we do that in other places, and generally _not_ relying on H2O act as a tool to provide test data would be a good idea (considering the fact that the intention of the test is to develop H2O).